silq/pulses/pulse_types.py

```diff
@@ -684,6 +683,11 @@ class FrequencyRampPulse(Pulse):
             initial frequency or stay at current frequency. Useful if the
             pulse doesn't immediately stop at the end (this depends on how
             the corresponding instrument/interface is programmed).
+        phase: Pulse phase. By default is set to zero.
+        phase_reference: What point in the the phase is with respect to.
+            Can be two modes:
+                - 'absolute': phase is with respect to `Pulse.t_start`.
```

Comment:
Absolute and relative are swapped here

Reply:
I copied this from SinePulse - do you have different definitions for different pulses?

Reply:
Swapped them and also for SinPulse as well